### PR TITLE
Add pagination to search page

### LINKED ISSUE #30: Search results are not paginated
currently only first 10 are being displayed..

See http://madewithwagtail.org/search/?q=blog for an example.

### PR DESCRIPTION
**Pull request by x-warrior**
Fix #30

### CODE REVIEW

File: core/templatetags/core_tags.py [hunk 1]
```diff
@@ -110,8 +110,13 @@ def subtract(value, arg):
...
-def str_tag(tag):
+@register.simple_tag
+def build_qsa(page_number, tag, query_string):
+    qsa = "?"
+    if page_number:
+        qsa += "page=%s&" % page_number
     if tag:
-        return "&tag=%s" % tag
-    return ""
+        qsa += "tag=%s&" % tag
```
Comment (thibaudcolas) [inline core/templatetags/core_tags.py]: This is adding one too many `&` at the end of the string – see for example http://localhost:8111/?page=2&tag=Blog&. 

I also think that this implementation is vulnerable to XSS injection – should it be using `urlencode` or some other form of escaping?

**Comment by thibaudcolas** [comment]
It works – but I think there are a couple things to fix, see details above.

**Comment by x-warrior** [comment]
@thibaudcolas I changed build_qsa method. Let me know what do you think.

**Comment by thibaudcolas** [comment]
Looking good, merged it now! There is another existing XSS vector that I'll fix separately.

**Pull request merged**